cmd/check/root.go

```diff
@@ -42,13 +44,35 @@ func newCheckCmd() *cobra.Command {
 				return err
 			}
 
+			subject := args[0]
+			relation := args[1]
+			namespace := args[2]
+			object := args[3]
+
+			var s *rts.Subject
+
+			// distinguish between subjectSet and subjectID
+			if strings.Contains(subject, "#") {
```

Comment:
I'd rather distinguish based on `:` here, since the relation could be empty (= wildcard) for a subject set, e.g., `check users:Foo access files README.md`.